### PR TITLE
fix: Add meta tags and preview image for SEO optimization

### PR DESCRIPTION
**Pull request by devXprite**
### ➕ Changes Introduced
Add meta tags for SEO optimization & add [this](https://user-images.githubusercontent.com/1811651/190090988-bd062b3d-635b-4ce1-ac15-81d3c9edcb91.png) img from Readme as  preview image for Twitter & Open Graph.

### 📷 Screenshots
**Before:**
In this Screenshot meta tags have default comments.
before

**After:**
after

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -10,7 +10,36 @@ export default function Home({ session }: PageProps) {
   return (
     <>
       <Head>
-        <title>{'defaang: recently-asked interview questions at FAANG+, collected & curated'}</title>
+        <title>Defaang</title>
```
Comment (ykdojo) [inline src/pages/index.tsx:13]: revert back to the original title

File: src/pages/index.tsx [hunk 2]
```diff
@@ -10,7 +10,36 @@ export default function Home({ session }: PageProps) {
...
+        <meta name='title' content='Defaang' />
+        <meta
```
Comment (ykdojo) [inline src/pages/index.tsx:15]: The metatags are in _document.tsx. Should it be fixed there instead?

File: src/pages/index.tsx [hunk 3]
```diff
@@ -10,7 +10,36 @@ export default function Home({ session }: PageProps) {
   return (
     <>
       <Head>
-        <title>{'defaang: recently-asked interview questions at FAANG+, collected & curated'}</title>
+        <title>Defaang</title>
+        <meta name='title' content='Defaang' />
```
Comment (ykdojo) [inline src/pages/index.tsx:14]: lowercase Defaang

**Review by ykdojo — CHANGES_REQUESTED** [review]

**Comment by devXprite** [comment]
@ykdojo I Did the suggested changes!!

**Review by ykdojo — APPROVED** [review]

**Pull request merged**